### PR TITLE
update packet and routing cmd

### PR DESCRIPTION
**Pull request by dgsbl**

### CODE REVIEW

File: modules/tibc/core/04-packet/types/errors.go [hunk 1]
```diff
@@ -10,24 +10,13 @@ const moduleName = host.ModuleName + "-" + SubModuleName
...
-	ErrInvalidChannel            = sdkerrors.Register(moduleName, 4, "invalid channel")
-	ErrInvalidChannelState       = sdkerrors.Register(moduleName, 5, "invalid channel state")
-	ErrInvalidChannelOrdering    = sdkerrors.Register(moduleName, 6, "invalid channel ordering")
-	ErrInvalidCounterparty       = sdkerrors.Register(moduleName, 7, "invalid counterparty channel")
-	ErrInvalidChannelCapability  = sdkerrors.Register(moduleName, 8, "invalid channel capability")
-	ErrChannelCapabilityNotFound = sdkerrors.Register(moduleName, 9, "channel capability not found")
 	ErrSequenceSendNotFound      = sdkerrors.Register(moduleName, 10, "sequence send not found")
 	ErrSequenceReceiveNotFound   = sdkerrors.Register(moduleName, 11, "sequence receive not found")
 	ErrSequenceAckNotFound       = sdkerrors.Register(moduleName, 12, "sequence acknowledgement not found")
 	ErrInvalidPacket             = sdkerrors.Register(moduleName, 13, "invalid packet")
```
Comment (chengwenxi) [inline modules/tibc/core/04-packet/types/errors.go]: ReIndex error msgs

File: modules/tibc/core/24-host/keys.go [hunk 2]
```diff
@@ -37,7 +37,7 @@ const (
 	KeyPacketCommitmentPrefix      = "commitments"
 	KeyPacketAckPrefix             = "acks"
 	KeyPacketReceiptPrefix         = "receipts"
-	KeyCleanPacketCommitmentPrefix = "cleans"
+	KeyCleanPacketCommitmentPrefix = "clean"
```
Comment (chengwenxi) [inline modules/tibc/core/24-host/keys.go:40]: 👍

File: modules/tibc/core/client/cli/cli.go [hunk 3]
```diff
@@ -1,6 +1,7 @@
 package cli
 
 import (
+	routing "github.com/bianjieai/tibc-go/modules/tibc/core/26-routing"
```
Comment (chengwenxi) [inline modules/tibc/core/client/cli/cli.go:4]: format

File: modules/tibc/core/04-packet/types/errors.go [hunk 4]
```diff
@@ -10,13 +10,13 @@ const moduleName = host.ModuleName + "-" + SubModuleName
...
-	ErrSequenceSendNotFound     = sdkerrors.Register(moduleName, 10, "sequence send not found")
-	ErrSequenceReceiveNotFound  = sdkerrors.Register(moduleName, 11, "sequence receive not found")
-	ErrSequenceAckNotFound      = sdkerrors.Register(moduleName, 12, "sequence acknowledgement not found")
-	ErrInvalidPacket            = sdkerrors.Register(moduleName, 13, "invalid packet")
-	ErrInvalidAcknowledgement   = sdkerrors.Register(moduleName, 16, "invalid acknowledgement")
-	ErrPacketCommitmentNotFound = sdkerrors.Register(moduleName, 17, "packet commitment not found")
-	ErrPacketReceived           = sdkerrors.Register(moduleName, 18, "packet already received")
-	ErrAcknowledgementExists    = sdkerrors.Register(moduleName, 19, "acknowledgement for packet already exists")
-	ErrInvalidCleanPacket       = sdkerrors.Register(moduleName, 21, "invalid clean packet")
+	ErrSequenceSendNotFound     = sdkerrors.Register(moduleName, 1, "sequence send not found")
```
Comment (chengwenxi) [inline modules/tibc/core/04-packet/types/errors.go:13]: Can not start from 1

**Review by chengwenxi — CHANGES_REQUESTED** [review]

**Review by chengwenxi — APPROVED** [review]

**Review by dreamer-zq — APPROVED** [review]

**Pull request merged**